### ISSUE TITLE
Error with summary() when using covariance structures with a random intercept

### ISSUE THREAD

**Issue by aosmith16** [glmmTMB/glmmTMB#291]
Upon updating **glmmTMB**/**TMB** to get current CRAN versions I'm getting an error message (seemingly about a mismatch in dimensions) when getting the summary() of a model using both a covariance structure and a random intercept.  

If I only use a covariance structure (no random intercept) things work as expected.

``` r
library(glmmTMB)
data(sleepstudy, package = "lme4")
sleepstudy$row <- factor(1:180)

# Model with random intercept and covariance structure
gt_min <- glmmTMB(Reaction ~ (1 | Subject) + ar1(row + 0 | Subject), sleepstudy)
summary(gt_min)  # Error message about number of columns
#>  Family: gaussian  ( identity )
#> Formula:          Reaction ~ (1 | Subject) + ar1(row + 0 | Subject)
#> Data: sleepstudy
#> 
#>      AIC      BIC   logLik deviance df.resid 
#>   1790.4   1806.3   -890.2   1780.4      175 
#> 
#> Random effects:
#> 
#> Conditional model:
#> Warning in base::rbind(..., deparse.level = deparse.level): number of
#> columns of result is not a multiple of vector length (arg 2)
#> Error in dimnames(x) <- dn: length of 'dimnames' [2] not equal to array extent

# Model wtih only covariance structure works
gt_min2 <- glmmTMB(Reaction ~ ar1(row + 0 | Subject), sleepstudy)
summary(gt_min2)
#>  Family: gaussian  ( identity )
#> Formula:          Reaction ~ ar1(row + 0 | Subject)
#> Data: sleepstudy
#> 
#>      AIC      BIC   logLik deviance df.resid 
#>   1789.0   1801.8   -890.5   1781.0      176 
#> 
#> Random effects:
#> 
#> Conditional model:
#>  Groups   Name Variance  Std.Dev.  Corr      
#>  Subject  row1 3.744e+03 6.119e+01 0.85 (ar1)
#>  Residual      1.162e-13 3.409e-07           
#> Number of obs: 180, groups:  Subject, 18
#> 
#> Dispersion estimate for gaussian family (sigma^2): 1.16e-13 
#> 
#> Conditional model:
#>             Estimate Std. Error z value Pr(>|z|)    
#> (Intercept)   301.33      11.03   27.33   <2e-16 ***
#> ---
#> Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1
```

**Comment by kaskr** [comment]
This used to work. Ran a quick `git bisect` and got:

92df55e228fde6ec5507ff4d4147eb162b10a61f is the first bad commit

**Comment by bbolker** [comment]
closed by  118f8fd
